src/this.js

```diff
@@ -14,18 +22,35 @@ class User {
 const me = new User({ username: 'LambdaSchool', password: 'correcthorsebatterystaple' });
 const result = me.checkPassword('correcthorsebatterystaple'); // should return `true`
 
+/*
+  console.log(me.username);
+  console.log(me.password);
+  console.log(result);
+*/
+
 const checkPassword = function comparePasswords(passwordToCompare) {
   // recreate the `checkPassword` method that you made on the `User` class
   // use `this` to access the object's `password` property.
   // do not modify this function's parameters
-  // note that we use the `function` keyword and not `=>`
+  // note that we use the `function` keyword and not `=>`{
+  if (passwordToCompare === this.password) {
+    return true;
+  }
+  return false;
 };
 
+
 // invoke `checkPassword` on `me` by explicitly setting the `this` context
 // use .call, .apply, and .bind
 
+checkPassword.call(this, me); // no idea!
+
 // .call
 
 // .apply
 
 // .bind
+
+/*
+  instructions are not explicit enough to answer this question without a test case to verify
```

Comment:
Check the solution branch for examples of how to solve this.  The key concept here is how the `this` context can be explicitly passed around.  MDN has some great resources on this